src/store/modules/contacts.js

```diff
@@ -132,45 +129,26 @@ export default {
     },
     async refresh ({ commit, rootGetters, getters }, addr) {
       // Make this generic over networks
+      const oldContactInfo = getters['getContact'](addr)
+      const updateInterval = getters['getUpdateInterval']
+      const now = moment()
+      if (oldContactInfo && moment(oldContactInfo.lastUpdate).add(updateInterval, 'milliseconds').isAfter(now)) {
+        // Short circuit if we already updated this contact recently.
+        console.log('skipping contact update, checked recently')
+        return
+      }
+      console.log('Updating contact', addr)
 
       // Get metadata
-      let handler = rootGetters['keyserverHandler/getHandler']
-      let relayData = null
       try {
-        let relayURL = await handler.getRelayUrl(addr)
-        let relayClient = new RelayClient(relayURL)
-        relayData = await relayClient.getRelayData(addr)
+        const handler = rootGetters['keyserverHandler/getHandler']
+        const relayURL = await handler.getRelayUrl(addr)
+        const relayClient = new RelayClient(relayURL)
+        const relayData = await relayClient.getRelayData(addr)
+        commit('updateContact', { addr, profile: relayData.profile, inbox: relayData.inbox })
       } catch (err) {
         console.error(err)
-        return
       }
-
-      let oldRelayData = getters['getContact'](addr)
-
-      // If profile values changed, then update
-      if (
```

Comment:
The intention here was to reduce the number of mutations caused by updates when there is no difference in the reported relay server data.

Removing this conditional (and the one below at 160) will cause excess mutations and hence more writes to disk.

Reply:
I've just made the following modifications to the `/profiles` method.

It's now possible to do add a query string `?digest=true` which responds with the sha256 digest of the profile metadata.

Reply:
We can implement in the future.  This is good for now.